src/document/json/text.ts

```diff
@@ -285,8 +290,9 @@ export class Text<A extends Indexable = Indexable> {
    */
   toString(): string {
     if (!this.context || !this.text) {
-      logger.fatal('it is not initialized yet');
-      return '';
+      const ErrorMessage = 'Text is not initialized yet';
+      logger.fatal(ErrorMessage);
+      throw new YorkieError(Code.ErrOperationNotReady, ErrorMessage);
     }
```

Comment:
@hackerwins 
The code seems to have become a bit verbose while separating the logger and throw error logic. There are places where only YorkieError is thrown, and others where the logger is also used.(e.g., it's different in toString and toJSON.) Is there a specific criterion for when to use `Logger.fatal`?

Reply:
I think the code is becoming verbose with duplicate logging and error throwing. After consideration, it seems better to simplify the approach:

1. Remove the Logger for now: This will reduce code duplication and complexity.
2. Keep only the 'throw new YorkieError' calls: This ensures consistent error handling.
3. Defer logger implementation: We can revisit and implement a comprehensive logging 
   strategy in a future PR.